susemanager-utils/susemanager-sls/salt/packages/profileupdate.sls

```diff
@@ -1,6 +1,7 @@
 packages:
   module.run:
     - name: pkg.info_installed
+{% if grains['os_family'] == 'Suse' %}
```

Comment:
This is maybe wrong. This 

```
attr: 'arch,epoch,version,release,install_date_time_t'
```

was AFAIK also ported to RedHat / yum

Reply:
Added `RedHat` to the condition.